ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -596,7 +596,9 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
             # HACK: try to "poke" the token endpoint before the test starts
             for i in $(seq 1 30); do
-              code="$( curl -s -o /dev/null -w "%{http_code}" https://oauth2.googleapis.com/token -X POST -d '' )"
+              if ! code="$( curl -s -o /dev/null -w "%{http_code}" https://oauth2.googleapis.com/token -X POST -d '')"; then
+                echo "Failed to POST https://oauth2.googleapis.com/token" 1>&2
```

Comment:
Write $? in this line

Reply:
https://github.com/openshift/release/pull/6190#discussion_r352929626